### ISSUE TITLE
LDAP

### ISSUE THREAD

**Issue by ozzyunder** [ngoduykhanh/wireguard-ui#195]
admin portal login and users access to download credentials via LDAP authentication would be great!

**Comment by fhughes90** [comment]
I agree with this request! Just added this container today to our lab infrastructure and first thing I thought of was LDAP integration.
